lib/A6sRailway.ts

```diff
@@ -92,14 +91,26 @@ export class A6sRailway {
      * @returns {Promise<A6sRailway>}
      */
     async execute(): Promise<A6sRailway> {
+        let workingDirectory = this.stationContext.getWorkingDirectory();
+
         if (!this.map) {
-            this.map = await this.a6sRailwayUtil.readYamlFile(this.mapFile);
-            const ctx = this.a6sRailwayUtil.getSharedContext();
-            ctx.pwd = dirname(this.mapFile);
+            try {
+                this.map = await this.a6sRailwayUtil.readYamlFile(
+                    this.a6sRailwayUtil.getAbsolutePath(this.mapFile, workingDirectory)
+                );
+                workingDirectory = dirname(this.mapFile);
```

Comment:
This may not work. You need to get dirname from absolute path in order to cover all scenarios